packages/stream/demo/index.tsx

```diff
@@ -0,0 +1,135 @@
+import { render } from "solid-js/web";
+import { Component, type JSX, createEffect, createSignal } from 'solid-js';
+import { createStream } from "../src";
+import { createPermission } from '../../permission/src'
+import { createCameras, createMicrophones } from "../../devices/src";
+
+import "uno.css";
+
+declare module "solid-js" {
+  namespace JSX {
+    interface ExplicitProperties {
+      srcObject?: MediaStream
+    }
+  }
+}
+
+const iceservers = {
+  iceServers: [
+    {
+      urls: ['stun:stun1.l.google.com:19302', 'stun:stun2.l.google.com:19302'],
+    },
+  ],
+  iceCandidatePoolSize: 10,
+}
+
+const App: Component = (): JSX.Element => {
+  const microphones = createMicrophones();
+  const cameras = createCameras();
+
+  const microphone = createPermission("microphone")
```

Comment:
using `microphone` as a name for the permission seems confusing, same for camera.